app/assets/stylesheets/dashboard.scss

```diff
@@ -112,11 +112,26 @@ table.dashboard {
     .detainee-container {
       height: 100%;
       margin-left: 1em;
-      border-left: 6px solid #990099;
       padding-right: 0;
       padding: 0px 0 10px 15px;
     }
```

Comment:
There shouldn't be anything visible that doesn't have a colour defined, I think it would be misleading to have a default.